framework/source/class/qx/core/Property.js

```diff
@@ -272,7 +272,8 @@ qx.Bootstrap.define("qx.core.Property",
       check        : null,       // Array, String, Function
       transform    : "string",   // String
       deferredInit : "boolean",  // Boolean
-      validate     : null        // String, Function
+      validate     : null,       // String, Function
+      comparator   : null        // String, Function
```

Comment:
Can we change this to a name that implies the meaning of the return value? Instead of comparator, how about isEqual or something of the sort?